packages/dd-trace/src/plugin_manager.js

```diff
@@ -134,9 +135,12 @@ module.exports = class PluginManager {
     }
 
     // TODO: update so that it's available for every CI Visibility's plugin
-    if (name === 'mocha') {
+    if (name === 'mocha' || name === 'jest') {
```

Comment:
Why is this needed actually? Could the option just be passed to everything like the other options? Then any plugin that supports agentless can see the config.

Reply:
yeah it could be. I thought it'd be best not to affect other plugins, but this is innocuous